app/views/vacancies/_sorting_options.html.haml

```diff
@@ -1,9 +1,10 @@
 .govuk-grid-row.sortable-links
   .govuk-grid-column-one-half
     = form_tag('', method: :get) do
-      - search_criteria.each do |filter, value|  
+      - search_criteria.each do |filter, value|
         - if filter == :phases
-          = hidden_field_tag "phases[]", value
+          - phases.each do | phase |
+            = hidden_field_tag "phases[]", phase
```

Comment:
[StackOverflow](https://stackoverflow.com/a/10343089) suggests you can use `multiple: true` config option to automagically add the `[]`.

Reply:
@dushan-madetech and I tried something similar to this.
Couldn't get it to work with `multiple: true`.

Reply:
I've also tried the examples in the link you suggested above.

Reply:
Would it be okay to get this merged this in since:
- It's behaving as desired
- Its other half is already in develop
?